.github/workflows/ci-build.yml

```diff
@@ -26,6 +26,11 @@ jobs:
           - webhook
     runs-on: ${{ matrix.os }}
     steps:
+      - name: Configure git settings (Windows)
+        if: matrix.os == 'windows-latest'
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
```

Comment:
This step is added to not convert files to ending lines with `\r\n`  after a checkout.

Some searching hints at `.gitattributes` being an option for converting files to `\n` endings before a commit, which might be an option if lintings begin to fail due to files being committed with `\r\n`!

Reply:
is there a reason why the files' ending lines are converted to this? I'm guessing we're getting rid of those ending lines because it's clashing with the linter's parsing?

Reply:
It seems to be a Windows default as far as I can tell 🤔

Some [suggestions](https://github.com/biomejs/biome/issues/4529#issuecomment-2474905129) add the following to `.gitattributes` to prefer `lf` but that might break editing experiences on certain setups that expect the `\r\n`:

```
* text eol=lf
```

And [some repos](https://github.com/golang/go/blob/5213e1e56c17bde612398dac8a2996bf020fe542/.gitattributes) treat text files as binary to never change endings on checkout:

```
* -text
```

That last example is so interesting to me. The linter does expect `\n` in all cases and was erroring if that wasn't found, and I'm hoping this change in CI is an alright enough workaround for matching expected line endings 👾